assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/inputs/PriceAmountInput.tsx

```diff
@@ -42,6 +43,7 @@ const PriceAmountInput: React.FC<PriceModifierProps> = ({ price }) => {
 				parse={formatParse()}
 				placeholder={__('amount...')}
 				setValue={setValue}
+				type='number'
```

Comment:
yeah, this is weird, I've reused `NumberInput` from chakra to avoid some boilerplate setup and to enhance number input, but have set type to text also on demo :man_shrugging:

Reply:
ya I was puzzled by it too, i likely should have taken this out since it didn't do anything